assets/js/modules/ads/components/settings/SettingsEdit.stories.js

```diff
@@ -16,14 +16,20 @@
  * limitations under the License.
  */
 
+import fetchMock from 'fetch-mock';
```

Comment:
I'll apply this myself.
```suggestion
/**
 * External dependencies
 */
import fetchMock from 'fetch-mock';
```